src/maintenance/azext_maintenance/generated/_params.py

```diff
@@ -0,0 +1,210 @@
+                   'recur. The rate can be expressed as daily, weekly, or monthly schedules. Daily schedule are '
+                   'formatted as recurEvery: [Frequency as integer][\'Day(s)\']. If no frequency is provided, the '
+                   'default frequency is 1. Daily schedule examples are recurEvery: Day, recurEvery: 3Days.  Weekly '
+                   'schedule are formatted as recurEvery: [Frequency as integer][\'Week(s)\'] [Optional comma '
+                   'separated list of weekdays Monday-Sunday]. Weekly schedule examples are recurEvery: 3Weeks, '
+                   'recurEvery: Week Saturday,Sunday. Monthly schedules are formatted as [Frequency as '
+                   'integer][\'Month(s)\'] [Comma separated list of month days] or [Frequency as '
+                   'integer][\'Month(s)\'] [Week of Month (First, Second, Third, Fourth, Last)] [Weekday '
+                   'Monday-Sunday]. Monthly schedule examples are recurEvery: Month, recurEvery: 2Months, recurEvery: '
+                   'Month day23,day24, recurEvery: Month Last Sunday, recurEvery: Month Fourth Monday.')
+
+    with self.argument_context('maintenance configuration update') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('resource_name', type=str, help='Resource Identifier')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('namespace', type=str, help='Gets or sets namespace of the resource')
+        c.argument('extension_properties', action=AddExtensionProperties, nargs='*', help='Gets or sets '
+                   'extensionProperties of the maintenanceConfiguration Expect value: KEY1=VALUE1 KEY2=VALUE2 ...')
+        c.argument('maintenance_scope', arg_type=get_enum_type(['All', 'Host', 'Resource', 'InResource', 'OSImage', ''
+                                                                'Extension', 'InGuestPatch', 'SQLDB', ''
+                                                                'SQLManagedInstance']), help='Gets or sets '
+                   'maintenanceScope of the configuration')
+        c.argument('visibility', arg_type=get_enum_type(['Custom', 'Public']), help='Gets or sets the visibility of '
+                   'the configuration')
+        c.argument('maintenance_window_start_date_time', type=str, help='Effective start date of the maintenance '
+                   'window in YYYY-MM-DD hh:mm format. The start date can be set to either the current date or future '
+                   'date. The window will be created in the time zone provided and adjusted to daylight savings '
+                   'according to that time zone.')
+        c.argument('maintenance_window_expiration_date_time', type=str, help='Effective expiration date of the '
```

Comment:
Define duplicated arguments as CLIArgumentType for reuse, please refer to other codes.